msteams-platform/bots/what-are-bots.md

```diff
@@ -46,17 +46,24 @@ You can use the Teams bot APIs to get information for members of a chat or team.
 <!--- TBD: For quick scanning, see if the above information can be itemized as a list.
 --->
 
-## Next step
+## Global endpoint for bots API
```

Comment:
This location doesn't feel right. This page talks about bots and what they do, and then here we pivot to a relatively obscure implementation detail, using terms that at this point they may not be familiar with yet.